### DISCUSSION TITLE
TimeThinker WS2

### DISCUSSION THREAD

**Discussion by sonicpixel2020** [RavenSystem/esp-homekit-devices#2057] [MEPLHAA Scripts]
Hi, after migrating to the new Format the Button on my outlet is not working.
Here are my configs:

Old Format, what was working with the power Button:
{"c":{"l":13,"i":0,"b":[{"g":4,"t":5}],"o":0},"a":[{"0":{"r":[{"g":14}]},"1":{"r":[{"g":14,"v":1}]},"t":2,"b":[{"g":4,"p":0}],"s":0}]}

new Format for HAA 12:
{"c":{"io":[[[13,14],2],[[4],6]],"o":0,"l":13,"i":1,"b":[[4,5]]},"a":[{"0":{"r":[[14]]},"1":{"r":[[14,1]]},"t":2,"b":[[4]],"s":0}]}

Can you please tell me what is wrong with the button configuration?

Here is a link to the outlet:
https://templates.blakadder.com/timethinker_WS2.html

Best Regards
Rene

**Comment by QDUNI** [comment]
Try this:

```
{"c":{"io":[[[13,14],2],[[4],6,1]],"o":0,"l":13,"i":1,"b":[[4,5]]},"a":[{"0":{"r":[[14]]},"1":{"r":[[14,1]]},"t":2,"b":[[4]],"s":0}]}
